src/Aspire.Hosting.AWS/SDKResourceExtensions.cs

```diff
@@ -46,6 +46,24 @@ public static IAWSSDKConfig WithRegion(this IAWSSDKConfig config, RegionEndpoint
         return config;
     }
 
+    /// <summary>
+    /// Disable validation for profile in sdk.
+    /// </summary>
+    /// <param name="config">An <see cref="IAWSSDKConfig"/> instance.</param>
+    public static IAWSSDKConfig WithNoValidation(this IAWSSDKConfig config)
+    {
+        if (config is AWSSDKConfig awsConfig)
```

Comment:
check my above comment - i think if we move that property to the IAWSSDKConfig interface then we wont need this cast. and the function can simply be

```
config.SDKValidationEnabled= enabled;
return config;
```